Meeting Minutes for May 16th (#26)
Cardano-Atlantic-Council/Minutes: 20250516.md

**Date:** May 16, 2025 19:00 UTC

**Attendees:** 
- Jenny Brito
- Beatrice Anihiri
- Thiago Nunes
- Adam Dean
- Mike Hornan
- Richard McCracken

**Purpose:** 
- [x] Assess and potentially vote on four current active Governance Actions

**Meeting Points:**

1. **Cardano Ecosystem Stablecoin DeFi Liquidity**

> The on-chain and off-chain hashes not matching was pretty straightforward. Discussions on any other aspects aside from the hash not matching that could cuase a vote of unconstitutional to knock that out while deliberating. The internal proposal committee managaement sections of the proposal plan to use on-chain voting and adjusted the threshold to a custome setting of 67% which could cause confustion.

 - Hash is the primary reason for unconstitinal
 - Changing a threshold in this unique manner is up for debate.
 - 6 of 6 team members voted No.
 - This proposal is determined to be unconstitutional.

2. **Cardano Blockchain Ecosystem Budget: Amaru Node Development 2025**
> All team members had analyzed this proposal and left comments for discussion in the chat asynchronously. Details were discussed, and no specific line items had been deemed unconstitutional.

>  - Made note of this budget has funding set aside for an unnamed future adminstriator and defines how the administrator will be acquired. Determined to be Ok although debatable.
>  - 5 of 6 voted Yes, 1 possible abstain pending further research.
>  - This proposal has exceeded our internal 4 of 6 threshold, determined to be constitutional.

Reviewer: ```suggestion
 - Made note of this budget has funding set aside for an unnamed future audito and defines how the auditor will be acquired. Determined to be OK although debatable.
 - 5 of 6 voted Yes, 1 possible abstain pending further research.
 - This proposal has exceeded our internal 4 of 6 threshold, determined to be constitutional.
```

3. **Set a 300 million ADA Net Change Limit for Epochs 563–635**
> Team members had analyzed this proposal and left comments for discussion in the chat asynchronously. Details were discussed, questions answered, and no specific line items had been found to be unconstitutional.

 - 6 of 6 team members voted Yes. Proposal is constitutional.

3. **2025 Cardano Blockchain Ecosystem Budget - 7.5M ₳ for community builders**

> > There are off-chain debates about which parties in this proposal actually want to receive funding support if the budget is approved. This proposal is a subset of many other proposal supported by the Intersect package the DReps voted on individually to rank each individual proposal submitted by developers. After further discusssion the team decided any off-chain discussions and ambiguities are not relevant to our decision making processs on the specific text of this proposal. 
> > Discussion about whether or not an adminstrator is actually specified, or management of funds, in the proposal.
> 
> 
>  - Proposal failed to address any type of administrator according to Article IV Section "shall" statement as a requirement states "Cardano Blockchain budgets shall specify a process for overseeing use of funds from Cardano Blockchain treasury withdrawals including designating one or more administrators who shall be responsible for such oversight."
>  - The abstract describes Intersect as an Administrator but does not appear to 
>  - 5 of 6 team members voted No, 1 voted abstain. Internal threshold met. Proposal is unconstitutional.

Reviewer: ```suggestion
> There are off-chain debates about which parties in this proposal actually want to receive funding support if the budget is approved. This proposal is a subset of many other proposal supported by the Intersect package the DReps voted on individually to rank each individual proposal submitted by developers. After further discussion the team decided any off-chain discussions and ambiguities are not relevant to our decision making process on the specific text of this proposal. 
> Discussion about whether or not an administrator is actually specified, or management of funds, in the proposal.

 - Proposal failed to address any type of administrator or process according to Article IV Section "shall" statement as a requirement states "Cardano Blockchain budgets shall specify a process for overseeing use of funds from Cardano Blockchain treasury withdrawals including designating one or more administrators who shall be responsible for such oversight."
 - The abstract describes Intersect as an Administrator but does not appear to 
 - 5 of 6 team members voted No, 1 voted abstain. Internal threshold met. Proposal is unconstitutional.
```

Bullet point #2: `The abstract describes Intersect as an Administrator but does not appear to...` === Incomplete thought/sentence?

**Action Items:**
1. Finish remaining research for #2 above and post decision async.

> 2. Draft rationales for all 4 decisions, and submit and final approval.

Reviewer: ```suggestion
2. Draft rationales for all 4 decisions, and submit for final approval.
```
